constructor/nsis/_nsis.py

```diff
@@ -280,7 +280,7 @@ def add_to_path(pyversion, arch):
 def rm_regkeys():
     cmdproc_reg_entry = NSISReg(r'Software\Microsoft\Command Processor')
     cmdproc_autorun_val = cmdproc_reg_entry.get('AutoRun')
-    conda_hook_regex_pat = r'((\s+&\s+)?\"[^\"]*?conda[-_]hook\.bat\")'
+    conda_hook_regex_pat = r'((\s+&\s+)?(if exist)?(\s?\"[^\"]*?conda[-_]hook\.bat\"))'
```

Comment:
```suggestion
    conda_hook_regex_pat = r'((\s+&\s+)?(if +exist)?(\s?\"[^\"]*?conda[-_]hook\.bat\"))'
```

I guess it doesn't hurt to allow more than one space in there?

Reply:
```suggestion
    conda_hook_regex_pat = r'((\s+&\s+)?(if +exist)?(\s*?\"[^\"]*?conda[-_]hook\.bat\"))'
```

Based on the regex pattern `r'(if exist \"[^\"]*?conda[-_]hook\.bat\" \"[^\"]*?conda[-_]hook\.bat\")'` in the init script, I don't foresee a situation in which multiple spaces would occur there, but it wouldn't hurt to allow for it.

In that case I'd also account for the possibility of multiple spaces between `exist` and the first `"`.